README.md

```diff
@@ -125,7 +127,8 @@ client = api.create(...)
 response = client.trading.orders.create(
     isin='US88160R1014',
     side='buy',
-    quantity=1
+    quantity=1,
+    venue='xmun',
```

Comment:
Is this required? it should be optional

Reply:
you are right, but in current form venue is passed as 'null'.  We should skip None fields while sending to the API -> this change will touch the code.